assembl/static2/css/components/assembl-glyph.scss

```diff
@@ -116,3 +116,5 @@
 .assembl-icon-ellipsis:before { content: '\f141'; } /* '' */
 .assembl-icon-ellipsis-vert:before { content: '\f142'; } /* '' */
 .assembl-icon-sentiment-neutral:before { content: '\e86f'; } /* '' */
+.assembl-icon-up-bold:before { content: '\e80e'; } /* '' */
+.assembl-icon-down-bold:before { content: '\e810'; } /* '' */
```

Comment:
please modify the hash in the urls to the font at the top of this file to avoid cache issue

Reply:
@paolina92 have you seen this one?

Reply:
@paolina92 please follow the doc `doc/fontello.rst`, you need to add new icons in the array in `styleguide/icons.jsx`
And be careful, @swergas modified the font too in #246